src/components/overflow-menu/overflow-menu.js

```diff
@@ -214,33 +241,56 @@ class OverflowMenu extends mixin(createComponent, initComponentBySearch, evented
     const key = event.which;
     const { element, optionMenu, options } = this;
     const isOfMenu = optionMenu && optionMenu.element.contains(event.target);
+    const isExpanded = this.element.getAttribute('aria-expanded') === 'true';
 
-    if (key === 27) {
-      this.changeState('hidden', getLaunchingDetails(event), () => {
-        if (isOfMenu) {
-          element.focus();
+    switch (key) {
+      // Esc
+      case 27:
+        this.element.removeAttribute('aria-expanded');
```

Comment:
Thought changing `aria-expanded`  is covered by: https://github.com/IBM/carbon-components/blob/956bfa94ec822b6f58bb5912cd43ed1e7d56618f/src/components/overflow-menu/overflow-menu.js#L144

Wondering if you saw otherwise?

Reply:
yes you are correct, this line is redundant